src/test/java/teammates/test/cases/ui/StudentFeedbackResultsPageActionTest.java

```diff
@@ -196,7 +196,7 @@ public void testExecuteAndPostProcess() throws Exception {
         long pageDataResultsVisibleFromTime = pageData.getBundle().feedbackSession.resultsVisibleFromTime.getTime();
         long dataBundleResultsVisibleFromTime = dataBundle.feedbackSessions.get("session1InCourse1")
                                                                            .resultsVisibleFromTime.getTime();
-        long TOLERANCE_TIME_IN_MILLISECONDS = 1000;
+        final long TOLERANCE_TIME_IN_MILLISECONDS = 1000;
```

Comment:
This one should be fine if fixed the other way around, i.e `long toleranceTimeInMs = 1000`.